lib/features/sign_up/view/sign_up_view.dart

```diff
@@ -63,9 +65,8 @@ class SignUpScreen extends StatelessWidget {
                           label: HatSpaceStrings.of(context).googleSignUp,
                           iconURL: Assets.images.google,
                           onPressed: () {
-                            context
```

Comment:
no, do not change this part, the new implementation is login on UI, which is not recommended